go-controller/pkg/cni/cni.go

```diff
@@ -258,6 +258,10 @@ func (pr *PodRequest) cmdAddWithGetCNIResultFunc(
 		}
 	} else {
 		response.PodIFInfo = podInterfaceInfo
+		if primaryUDNPodRequest != nil {
```

Comment:
above on line 220 we have this comment:

		//TODO: There is nothing technical to run this at unprivileged mode but
		//      we will tackle that later on.			//      we will tackle that later on

This comment was about making unpriveleged mode work with PUDN...obviously our comment kind of sucked :) Can you please remove this comment in your commit as well?

Reply:
fixed